rust-tvtf/src/lib.rs

```diff
@@ -74,6 +74,34 @@ pub fn get_function_registries() -> anyhow::Result<Vec<FunctionRegistry>> {
             )
             .build()
             .context("create `flatten` registry failed")?,
+        FunctionRegistry::builder()
```

Comment:
Although these lines are actually unused in our current implementation (the function resolver only uses the definitions in our SQL metastore currently), I'd like to remind you that it's not necessary to write `FunctionRegistry::builder()` twice. you could simply add another `.signature()` call.